controllers/cassandrarestore_controller.go

```diff
@@ -328,8 +366,21 @@ func isCassdcReady(cassdc *cassdcapi.CassandraDatacenter) bool {
 	return statusReady == corev1.ConditionTrue
 }
 
+func isCassdcUpdating(cassdc *cassdcapi.CassandraDatacenter) bool {
```

Comment:
This part of the code is so often replicated in every operator, that we might want to push it to the cassdcapi.